app/json/slate_configs.json

```diff
@@ -89,6 +89,24 @@
       }
     ]
   },
+  {
+    "id": "f99178fb-6bd0-4fa1-8109-cda181b697f6",
+    "displayName": "New Tab Recommendations",
+    "description": "A selection of content for display on the Firefox new tab",
+    "internalDescription": "en-us New Tab recommendation items for today",
+    "experiments": [
+      {
+        "description": "default",
+        "candidateSets": [
+          "493a5556-9800-449f-8f8c-c27bb6c8c810"
+        ],
+        "rankers": [
+          "top30-items",
+          "thompson-sampling-28day"
```

Comment:
you can remove thompson-sampling if you want predictable item ordering within the slate